runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/artifact/ArtifactNotFoundException.java

```diff
@@ -0,0 +1,19 @@
+/*
```

Comment:
Remove this file?

Reply:
Yep.  I ended up making it a nested class inside DfsArtifactRetrievalService, but looks like I never cleaned up the file.